customer/.env.production

```diff
@@ -1,5 +1,6 @@
-NEXT_PUBLIC_API_URL=https://customer.api.helpwave.de
-NEXT_PUBLIC_OIDC_PROVIDER=https://id.helpwave.de/realms/main/
-NEXT_PUBLIC_CLIENT_ID=customer-api
-NEXT_PUBLIC_REDIRECT_URI=https://customer.helpwave.de
-NEXT_PUBLIC_POST_LOGOUT_REDIRECT_URI=https://customer.helpwave.de/
+NEXT_PUBLIC_API_URL=https://api.customer.helpwave.de
+NEXT_PUBLIC_OIDC_PROVIDER=https://id.helpwave.de/realms/main
+NEXT_PUBLIC_CLIENT_ID=customer
```

Comment:
Suggestion: use fqdn to improve identification oft clients

Reply:
Ty, good advise. I'll update it.